source/elements/oneMKL/source/domains/spblas/data_types/set_matrix_property.rst

```diff
@@ -0,0 +1,95 @@
+.. SPDX-FileCopyrightText: 2024 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_sparse_set_matrix_property:
+
+set_matrix_property
+===================
+
+Matrix properties
+-----------------
+
+.. container:: section
+
+   .. code:: cpp
+
+      enum class matrix_property {
+          symmetric,
+          sorted,
+      };
+
+   TODO: Clarify are the properties meant to be used as a bit mask so multiple
+   properties can be set?
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 20 80
+
+      * - Value
+        - Description
+      * - ``symmetric``
+        - Guarantees that the matrix is symmetric, meaning it must be square and
+          its transpose is equal to itself. TODO: Confirm whether the user can
+          only provide half of the data in this case?
+      * - ``sorted``
+        - Guarantees that the column indices are sorted for a given row. Only
+          available for the CSR format. TODO: Confirm the description.
```

Comment:
@gajanan-choudhary, I could use your advice to improve this description for what you were planning exactly.

Reply:
Clarified this in https://github.com/oneapi-src/oneAPI-spec/pull/522/commits/b4c2a467f235d21df79e7efd8216e665d4357fdf